apps/patient/src/components/PickupOptions.tsx

```diff
@@ -11,11 +19,14 @@ export const PickupOptions = ({
   handleShowMore,
   loadingMore,
   showingAllPharmacies,
-  isMobile
+
```

Comment:
empty space, do you have prettier on save?

Reply:
yeah, it'll leave space though

Reply:
removed